Workflow.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+var rightAngle =function(size){
+    var stars = '';
+    for(var i = 0; i < size; i++){
+        stars += '*';
+        console.log(stars);
+    }
+}
+
+rightAngle(3);
+rightAngle(5);
+
+var equalateralTriangle= function(size){
+    var stars = '';
+    var spaces = '';
+    for(var i = 0 ; i < size; i++){
+        spaces = '';
+        for(var j = size -1-i; j > 0; j--) {
+            spaces += ' ';
+        }
+        stars += '*';
+        if(stars.length > 1) {
+            stars += '*'
+        }
+        console.log(spaces+stars);
+    }
+}
+
+equalateralTriangle(12);
+
+var fibonacciArray=[0,1];
+
+for(var i = 1; i < 10; i++) {
+    var result =fibonacciArray[i-1]+fibonacciArray[i];
+    fibonacciArray.push(result);
+}
+console.log(fibonacciArray);
+
+
+for(var i= 1; i<=100; i++){
+    if (i%3===0 && i%5!=0){
```

Comment:
Going to give you a shipit, barring any other problems, but there is a simpler way:
Output JavaScript % 5 and % 3
ELSE IF %3 output Java
ELSE IF %35output Script

There are several solutions to this problem, all valid, all with deficiencies - which is why I set it.